hack/tools/test_module.sh

```diff
@@ -0,0 +1,201 @@
+# kubectl apply -f /data/checkagain14/data-movement-operator/modules/implicit-copy-batch-module.yaml -n fybrik-system
+
+
+#datashim
+bin/kubectl apply -f https://raw.githubusercontent.com/datashim-io/datashim/master/release-tools/manifests/dlf.yaml
+bin/kubectl wait --for=condition=ready pods -l app.kubernetes.io/name=dlf -n dlf --timeout=500s
+# cd $PATH_TO_LOCAL_FYBRIK/third_party/datashim/
+# make deploy
+
+
+
+# source ${EXPORT_FILE}
+
+
+
+# Notebook sample
+
+bin/kubectl create namespace fybrik-notebook-sample
+bin/kubectl config set-context --current --namespace=fybrik-notebook-sample
+
+#localstack
+bin/helm repo add localstack-charts https://localstack.github.io/helm-charts
+bin/helm install localstack localstack-charts/localstack --set startServices="s3" --set service.type=ClusterIP
+bin/kubectl wait --for=condition=ready --all pod -n fybrik-notebook-sample --timeout=400s
+
+bin/kubectl port-forward svc/localstack 4566:4566 &
+
+export ENDPOINT="http://127.0.0.1:4566"
+export BUCKET="demo"
+export OBJECT_KEY="PS_20174392719_1491204439457_log.csv"
+export FILEPATH="$WORKING_DIR/PS_20174392719_1491204439457_log.csv"
+aws configure set aws_access_key_id ${ACCESS_KEY} && aws configure set aws_secret_access_key ${SECRET_KEY} && aws --endpoint-url=${ENDPOINT} s3api create-bucket --bucket ${BUCKET} && aws --endpoint-url=${ENDPOINT} s3api put-object --bucket ${BUCKET} --key ${OBJECT_KEY} --body ${FILEPATH}
+
+
+cat << EOF | bin/kubectl apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: paysim-csv
+type: Opaque
+stringData:
+  access_key: "${ACCESS_KEY}"
+  secret_key: "${SECRET_KEY}"
+EOF
+
+
+bin/kubectl apply -f $WORKING_DIR/Asset-$moduleVersion.yaml -n fybrik-notebook-sample
+
+
+
+
+#fybrikstorage
+cat << EOF | bin/kubectl apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: bucket-creds
+  namespace: fybrik-system
+type: Opaque
+stringData:
+  access_key: "${ACCESS_KEY}"
+  accessKeyID: "${ACCESS_KEY}"
+  secret_key: "${SECRET_KEY}"
+  secretAccessKey: "${SECRET_KEY}"
+EOF
+
+bin/kubectl apply -f $WORKING_DIR/fybrikStorage-$moduleVersion.yaml -n fybrik-system
+
+
+bin/kubectl -n fybrik-system create configmap sample-policy --from-file=$WORKING_DIR/sample-policy-$moduleVersion.rego
+bin/kubectl -n fybrik-system label configmap sample-policy openpolicyagent.org/policy=rego
+
+c=0
+while [[ $(bin/kubectl get cm sample-policy -n fybrik-system -o 'jsonpath={.metadata.annotations.openpolicyagent\.org/policy-status}') != '{"status":"ok"}' ]]
+do
+    echo "waiting"
+    ((c++)) && ((c==25)) && break
+    sleep 5
+done
+
+
+bin/kubectl apply -f $WORKING_DIR/fybrikapplication-$moduleVersion.yaml
+
+c=0
+while [[ $(bin/kubectl get fybrikapplication my-notebook -o 'jsonpath={.status.ready}') != "true" ]]
+do
+    echo "waiting"
+    ((c++)) && ((c==30)) && break
+    sleep 6
+done
+
+bin/kubectl get pods -n fybrik-blueprints
+POD_NAME=$(bin/kubectl get pods -n fybrik-blueprints -o=name | sed "s/^.\{4\}//")
```

Comment:
There are two pods in fybrik-blueprints: one for the fligh module and the other for DMO, thus label selector can be chosen to select the flight module:
`POD_NAME=$(bin/kubectl get pods  -l app.kubernetes.io/name=arrow-flight-module-chart -n fybrik-blueprints -o=name | sed "s/^.\{4\}//")`